### ISSUE TITLE
Alias may need further thought

### ISSUE THREAD

**Issue by systep** [ConsenSys/real-estate-standards#5]
One options is to keep it very simple, and just be able to add/remove (as is with current implementation). However, I think we can be more informative here, and suggest a field AliasType and AliasSource which could indicate the purpose. 

This could help with Non-addressable Place Names (NAPNs) - e.g. Alias='Central Park'; AliasType='NAPN'; AliasSource='NYC_DCP_PlaceNamesDirectory'; another example: Alias='220 Riverside Blvd., #13F'; AliasType='Postal'; AliasSource='USPS_V13';

This would also help with identifying the "primary" alias, i.e. the known street name - very helpful for practical implementation when geohash won't be perfect... e.g. Alias='220 Riverside Boulevard, #13F'; AliasType='PrimaryPostal'; AliasSource='NYC_SND'

Also addresses ranges (and continuous parity - i.e. numbers are not odd/even):
Alias='210 Riverside Blvd.'; AliasType='LowPostalRange'; AliasSource='NYC_DCP_SND'
Alias='240 Riverside Blvd.'; AliasType='HighPostalRange'; AliasSource='NYC_DCP_SND'
Alias='YES'; AliasType='PostalRangeContinuousParityIndicator'; AliasSource='NYC_DCP_SND'

Bottom line, I think we can be more granular with alias (probably not the best implementation above, but should point us in the right direction to start thinking about it)
